values/harbor/harbor.gotmpl

```diff
@@ -6,13 +6,13 @@
 {{- $harborSecretName := ($harborDomain | replace "." "-") }}
 {{- $notarySecretName := ($notaryDomain | replace "." "-") }}
 {{- $externalUrl := printf "https://%s" $harborDomain }}
-{{- $tag := $h | get "image.tag" "v2.4.0" }}
+{{- $tag := $h | get "image.tag" "v2.5.4" }}
 
 externalURL: {{ $externalUrl }}
 fullnameOverride: harbor
 # logLevel - debug, info, warning, error or fatal
 logLevel: warning
-harborAdminPassword: {{ $h | get "adminPassword" $v.otomi.adminPassword }}
```

Comment:
I remember that you mentioned that there was an issue with logic here causing harbor db password being taken from the Helm chart. Still, I am wondering if this change can have any side effects on existing deployments. If so then we need to include it in the release notes.